src/components/App/ActiveShowModal/index.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { hideDetails } from 'redux/actions/detailsActions';
+
+import imgMissing from 'images/img_missing.png';
+
+import styles from './ActiveShowModal.module.scss';
+
+class ActiveShowModal extends Component {
```

Comment:
I like the idea of modal with details, but as you wrote - it allows to preview details only for one episode at the time. Which is also possible with pure React solution. I still think that you should stay with pure React but do a modal style details when user click on the poster. And add new button (except the one in the modal) to go to the Show.